src/github/logic.py

```diff
@@ -218,7 +219,9 @@ def pull_request_participants(pull_request: PullRequest) -> List[str]:
             for gh_handle in (
                 [pull_request.author_handle()]
                 + pull_request.assignees()
-                + pull_request.requested_reviewers()
+                + pull_request.requested_reviewers(
+                    include_team_members=not SGTM_FEATURE__DISABLE_GITHUB_TEAM_SUBSCRIPTION
+                )
                 + _pull_request_body_mentions(pull_request)
```

Comment:
This is interesting. This change leaves CCed reviewers intact. Creating PRs using git pr currently this is what I see:

* if I select a team in the first prompt for reviewer, it picks a random member from the team as the reviewer. like [this](https://github.com/Asana/codez/pull/198536). 
* if I select a team in the second prompt for reviewer, it adds every member of the team, but ALSO adds them as cced like [this](https://github.com/Asana/codez/pull/198536) (which your pr doesn't block notifications for. 

I think the git PR flow would also need to be updated to 
1. take the whole team as the first reviewer prompt
2. not add all reviewers as CCed (or also update this PR to exclude CCed people). 

Another note: I would couple this with [github notifications ](https://app.asana.com/0/1193066022368038/1202528365393009/f)and you're golden!

Reply:
Ah, you might be a little confused. I made two separate changes:
* A change to Asana's internal git pr flow that enables you to add an actual github team as a reviewer or cc (see https://github.com/Asana/codez/pull/195892) if the team you’re trying to use isn’t in the hardcoded allowlist you’ll get the legacy behavior of just adding all the team members as direct assigns or CCs
* SGTM separately currently denormalizes team assigns (which before the previous change never happened anyways within Asana) and added everyone as a follower on the task. My change here allows you to disable that functionality